api/server.go

```diff
@@ -53,20 +57,20 @@ func NewServer(conf ServerConfig) (*standard.Server, error) {
 	environments := NewEnvironmentResource(conf.Store, conf.Stats)
 	users := NewUserResource(conf.Store, conf.Stats)
 
-	e.Get("/api/health", standard.WrapHandler(healthz.Handler()))
+	e.Get(os.Getenv("PREFIX_ROUTE")+"/api/health", standard.WrapHandler(healthz.Handler()))
```

Comment:
Instead of getting the env variable here, add the prefix to the `ServerConfig` struct.